README.md

```diff
@@ -1,22 +1,24 @@
+[![Travis](https://img.shields.io/travis/numpy/numpy.svg)](https://travis-ci.org/numpy/numpy) 
+
```

Comment:
When rendered by Github, it will show the current Travis build status of the project. http://shields.io/